contracts/payloads/IGP89/PayloadIGP89.sol

```diff
@@ -390,20 +354,19 @@ contract PayloadIGP89 is PayloadIGPMain {
         }
 
         {
-            // [TYPE 2] LBTC-cbBTC<>WBTC vault
-            VaultConfig memory VAULT_LBTC_cbBTC__WBTC = VaultConfig({
-                vault: LBTC_cbBTC__WBTC_VAULT,
-                vaultType: VAULT_TYPE.TYPE_2,
-                supplyToken: address(0),
-                borrowToken: WBTC_ADDRESS,
-                baseWithdrawalLimitInUSD: 0,
-                baseBorrowLimitInUSD: 10_000, // $10K
-                maxBorrowLimitInUSD: 15_000 // $15K
+            // Update LBTC-cbBTC<>cbBTC vault supply shares limit
+            IFluidAdminDex.UserSupplyConfig[]
+                memory config_ = new IFluidAdminDex.UserSupplyConfig[](1);
+            config_[0] = IFluidAdminDex.UserSupplyConfig({
+                user: LBTC_cbBTC_DEX,
+                expandPercent: 35 * 1e2, // 35%
+                expandDuration: 6 hours, // 6 hours
+                baseWithdrawalLimit: 110 * 1e8 // 110 shares
```

Comment:
should be dust limit. 1 share = ~162k so 0.05 shares


![image](https://github.com/user-attachments/assets/ab11ba5d-6fba-4cc5-bb78-81de40db2a31)